cmd/keytransparency-signer/backend.go

```diff
@@ -132,7 +133,11 @@ func main() {
 	}
 	mutator := entry.New()
 
+	if *maxEpochDuration < *minEpochDuration {
+		glog.Exitf("Maximum time between epoch creation (%v) must be grater or equal minimum time between epoch creation.")
```

Comment:
It's only required for tests, but it's useful info to have for other failures too because it also gives the desired value.